app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,59 @@
+class PasswordResetsController < ApplicationController
+  before_action :get_user, only: %i(edit update)
+  before_action :valid_user, only: %i(edit update)
+  before_action :check_expiration, only: %i(edit update)
+
+  def new; end
+
+  def edit; end
+
+  def create
+    @user = User.find_by email: params[:password_reset][:email].downcase
```

Comment:
```suggestion
    @user = User.find_by email: params.dig(:password_reset, :email)&.downcase
```
tìm hiểu method `dig` nhé

Reply:
dig là method tìm kiếm với hash và rút gọn của cách dùng toán tử &&. Nó sẽ thực hiện tìm kiếm với lần lượt từng key và trả về nil ngay nếu có 1 key trả về kết quả nil.